docs/guide/drag_and_drop.md

```diff
@@ -0,0 +1,36 @@
+# Drag and drop interface class
```

Comment:
Maybe something like "Creating drag and drop elements", to align with other cookbook titles?